scripts/effects/sigil.lua

```diff
@@ -60,46 +33,21 @@ effectObject.onEffectLose = function(target, effect)
     local power = effect:getPower() -- Tracks which bonus effects are in use.
     -- local subPower = effect:getSubPower() -- subPower sets % required to trigger regen/refresh.
 
-    if
-        power == 1 or
-        power == 3 or
-        power == 5 or
-        power == 7 or
-        power == 9 or
-        power == 11 or
-        power == 13 or
-        power == 15
-    then
+    if utils.mask.getBit(power, 1) then
         local percentage = 70 -- TODO: This should be based off of controlled areas in Campaign
         target:delLatent(xi.latent.SIGIL_REGEN_BONUS, percentage, xi.mod.REGEN, 1)
     end
 
-    if
-        power == 2 or
-        power == 3 or
-        power == 6 or
-        power == 7 or
-        power == 10 or
-        power == 11 or
-        power >= 14
-    then
+    if utils.mask.getBit(power, 2) then
         local percentage = 60 -- TODO: This should be based off of controlled areas in Campaign
         target:delLatent(xi.latent.SIGIL_REFRESH_BONUS, percentage, xi.mod.REFRESH, 1)
     end
 
-    if
-        effect:getPower() >= 4 and
-        effect:getPower() <= 7
-    then
-        target:delMod(xi.mod.FOOD_DURATION, 100)
-    elseif
-        effect:getPower() >= 8 and
-        effect:getPower() <= 11
-    then
-        -- target:delMod(xi.mod.EXPLOSS_REDUCTION), ???)
-        -- exp loss reduction not implemented.
-    elseif effect:getPower() >= 12 then
+    if utils.mask.getBit(power, 3) then
         target:delMod(xi.mod.FOOD_DURATION, 100)
+    end
+
+    if utils.mask.getBit(power, 4) then
         -- target:delMod(xi.mod.EXPLOSS_REDUCTION), ???)
         -- exp loss reduction not implemented.
```

Comment:
Do we really not have exp loss reduction? Isn't there a moghancement that does this?

Reply:
EXPERIENCE_RETAINED, I think.

Reply:
Comment in code was written many many years ago and when mod was created, this didn't get updated. So thats another easy PR for somebody.